product_docs/docs/biganimal/release/getting_started/identity_provider/index.mdx

```diff
@@ -78,23 +78,22 @@ Once your identity provider is set up, you can view your connection status, ID,
 You need a verified domain so your users can have a streamlined login experience with their email address. 
 
 1. On the **Domains** tab, enter the domain name and select **Next: Verify Domain**. 
-2. Copy the TXT record and follow the instructions in the on screen verify box (repeated below), to add it as a TXT record on that domain within your DNS provider's management console.
+2. To add it as a TXT record on that domain in your DNS provider's management console, copy the TXT record and follow the instructions in the on-screen verify box:
 
-   - Log in to your domain registrar or web host account.
-   - Navigate to the DNS settings for the domain you want to verify.
-   - Add a TXT record.
-   - In the Name field, enter @.
-   - In the Value field, enter the verification string provided, eg. 
-   - “edb-biganimal-verification=VjpcxtIC57DujkKMtECSwo67FyfCExku”
-   - Save your changes and wait for the DNS propagation period to complete. This can take up to 48 hours.
+   1. Log in to your domain registrar or web host account.
+   1. Navigate to the DNS settings for the domain you want to verify.
+   1. Add a TXT record.
+   1. In the **Name** field, enter `@`.
+   1. In the **Value** field, enter the verification string provided, for example, `"edb-biganimal-verification=VjpcxtIC57DujkKMtECSwo67FyfCExku"`.
```

Comment:
Is this supposed to have the quotes? It had quotes in the original, and I wanted to just tag it as code, but I wanted to verify the string does include the quote marks. @edcrewe